packages/web-components/src/components/cta/card-cta.ts

```diff
@@ -219,5 +221,9 @@ class DDSCardCTA extends VideoCTAMixin(CTAMixin(DDSCard)) {
   static styles = styles; // `styles` here is a `CSSResult` generated by custom WebPack loader
 }
 
-/* @__GENERATE_REACT_CUSTOM_ELEMENT_TYPE__ */
-export default DDSCardCTA;
+export default deprecate(
+  /* @__GENERATE_REACT_CUSTOM_ELEMENT_TYPE__ */
+  DDSCardCTA,
+  `The dds-card-cta component has been deprecated. All its features have been absorbed into 
+  the base dds-card component. See migration guide for more information.`
+);
```

Comment:
I think @ariellalgilmore and I ran into build errors when we tried to use the `deprecate` utility on web components. We may have done something else instead to avoid the build errors

Reply:
oh yeah! the `deprecate` utility does not work with react wrappers. We instead just added `console.warn` statements instead like the following: https://github.com/carbon-design-system/carbon-for-ibm-dotcom/blob/feat/carbon-for-ibm-dotcom-v2/packages/web-components/src/components/pictogram-item/pictogram-item.ts#L61-L64